### DISCUSSION TITLE
url white list match domain instead of exact string

### DISCUSSION THREAD

**Discussion by odama626** [directus/directus#5662] [Feature Requests]
Started messing with the user invite flow and saw that the url whitelist only works for exact matches.  would it make more sense to match the domain?

https://github.com/directus/directus/blob/9e93342af0a6b0689a6d23ea144a02eb1940d372/api/src/services/users.ts#L231

what if I want to do something like this?
```js
`https://example.com/invite?from=${user.id}&event=${event.id}&to=${email}`
```

maybe a regex?
```js
if (url && urlWhitelist.includes(url.match(/\/\/(.*?)\//)?.[1]) === false) {
```
that would match `example.com`

**Comment by rijkvanzanten** [comment]
> would it make more sense to match the domain?

It would!

Wanna open a PR for that? 🙂

**Reply by odama626** [reply]
sure https://github.com/directus/directus/pull/5694
